scripts/azure-github-comment.js

```diff
@@ -33,10 +33,8 @@ async function withGithub(url, json, method) {
 
   // Find my comment
   const updateComment = comments.find(({ body }) => body.includes(REPLACE_MARK));
-  console.log('Origin comment:', updateComment);
 
   // Update
-  let res;
```

Comment:
Why delete this?

Reply:
Please check again.

Reply:
`components/carousel/index.tsx`
  114:7  error  Unexpected string concatenation  prefer-template

Please use ES6 template string.